### PR TITLE
add configure hook

### PR DESCRIPTION
**Pull request by lucyllewy**
also add minor changes to snapcraft.yaml for correctness
move the app into /opt/tvhproxy to keep things nicely organised
replace manual python-packages with utilising the requirements.txt from upstream

### CODE REVIEW

File: snap/snapcraft.yaml [hunk 1]
```diff
@@ -18,14 +18,11 @@ parts:
...
     source-type: git
-    python-packages:
-      - flask
-      - requests
-      - gevent
+    requirements: requirements.txt
     install: |
-      cp tvhProxy.py $SNAPCRAFT_PART_INSTALL
-      cp -r templates $SNAPCRAFT_PART_INSTALL
+      cp  -R ./ $SNAPCRAFT_PART_INSTALL/opt/tvhproxy
```
Comment (morphis) [inline snap/snapcraft.yaml]: What is the reason to move it into $SNAP/opt?

Comment (lucyllewy) [reply snap/snapcraft.yaml]: That was an aesthetic change to tidy up the filesystem.

Comment (morphis) [reply snap/snapcraft.yaml]: I would prefer, if we move it, to move it to `$SNAP/bin`. IMHO `/opt` doesn't make much sense inside a snap.

**Comment by morphis** [comment]
One small comment in line, otherwise LGTM

**Comment by lucyllewy** [comment]
I've moved the files back where they were originally reverting my change that put them into `$SNAP/opt`.

**Comment by morphis** [comment]
LGTM, thanks for doing the changes!

**Pull request merged**